.github/workflows/release.yaml

```
@@ -0,0 +1,25 @@
+# build and release new docs as a release asset
+name: Release Docs
```

I've become a fan of emojis on workflow names now; maybe:

```suggestion
name: 📦 Package Docs
```

(I'm calling it "Package" in favor of "Release", since it's not technically releasing anything.)